src/piracy.jl

```diff
@@ -20,3 +19,29 @@ end
 function OpenAPI.from_json(a::Vector{IPUMS.Shapefile}, b)
     [IPUMS.Shapefile(s["name"], s["year"], s["geographicLevel"], s["extent"], s["basis"], s["sequence"]) for s in b["data"]]
 end
+
+"""
+
+This is a pirated method that supports the `extract_list` method in returning additional information about `page_size`, `page_number`, and generated URLs.
```

Comment:
Should we mention what the resolution to this issue is, in the comment. What does an unpirated version of the method involve? Just to make sure that we know how to fix the problem when we return to it later.

Reply:
Good idea. I'll add a comment in the docstring.